data/base-crafting.yaml

```diff
@@ -467,16 +467,20 @@ tailoring:
     - 16661
     - 16663
     sewing-rooms:
-    - 19_037
-    - 19_036
-    - 16_670
+    - 19037
```

Comment:
The underscore here is a Rubocop style thing.

Which reminds me, we should probably run Rubocop on the repo sometime soon. I just don't have it/Ruby installed on any machines at the moment.